docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,229 @@
-# Configure a new project
+# Tools to customise a new Kedro project
 
-<!--TO DO-->
-<!--Detailed usage of add-ons goes here-->
+There are several ways to configure your new project with the tools and example code.
+
+## `kedro new`
+
+Navigate to the directory in which you would like to create your new Kedro project, and run the following command:
+
+```bash
+kedro new
+```
+
+This will start the new project creation workflow.
+
+``` {note}
+You can also add flags to `kedro new` to skip some or all of the steps in the project creation workflow to skip queries about how you want to customise the project. The flags are described below.
+```
+
+### Project name
+The first prompt asks you to input a project name.
+
+You can skip the step to name the project by adding it to your command. For example:
+
+```bash
+kedro new --name=spaceflights
+```
+
+### Tools
+You are then asked to select which tools to include. Choose from the list using comma separated values `(1,2,4)`, a range of values `(3-5)`, or the key words `all` or `none`. Skipping the prompt by entering no value will result in the default selection of `none`. Further information about each of the tools is described below in [Kedro tools](#kedro-tools).
+
+
+| ![](../meta/images/kedro-new-tools-prompt.png)|
+|:--:|
+| *Prompt for tools selection* |
+
+
+You may also specify your tools selection directly from the command line by using the flag `--tools`:
+
+```bash
+kedro new --tools=<your tool selection>
+```
+
+To specify your desired tools you must provide them by name as a comma separated list, for example `--tools=lint,test,viz`. The following tools are available for selection: `lint`, `test`, `log`, `docs`, `data`, `pyspark`, and `viz`.
+
+A list of available tools can also be accessed by running `kedro new --help`
+
+![](../meta/images/kedro-new-help.png)
```

Comment:
In this screenshot there's still a mention of `Pyspark` with lower case `s`. I think we solved that right? So let's also update the screenshot.

Reply:
Addressed in [9eaebe6](https://github.com/kedro-org/kedro/pull/3331/commits/9eaebe68ed008d1b7db6a5cf2673a797a31b176b)